--- nxserver.old	2004-10-14 18:57:21.000000000 +0200
+++ nxserver	2004-10-14 18:57:24.000000000 +0200
@@ -19,10 +19,10 @@
 # do not use any bash scripting in this paths!
 # (sourced by other scripts via export)
 
-NX_DIR=/usr/
-NX_SESS_DIR=/var/lib/nxserver/
-NX_ETC_DIR=/etc/nxserver/
-NX_HOME_DIR=/home/.nx/
+NX_DIR=/usr/NX
+NX_SESS_DIR=/usr/NX/var/db
+NX_ETC_DIR=/usr/NX/etc
+NX_HOME_DIR=/usr/NX/home/nx
 
 NX_VERSION=1.4.0-02
 NX_LICENSE="OS_(GPL)"
--- nxnode.old	2004-10-14 18:57:21.000000000 +0200
+++ nxnode	2004-10-14 18:57:24.000000000 +0200
@@ -162,7 +162,7 @@
 		P=""
 		[ -n "$agent_user" ] && U="-u $agent_user"
 		[ -n "$agent_password" ] && P="-p $agent_password"
-		$NX_DIR/bin/nxdesktop -name "NX - $user@$HOSTNAME:$display - $session (GPL Edition)" $K $G $U $P $agent_server 2>>~/.nx/C-$sess_id/session &
+		LD_LIBRARY_PATH=$NX_DIR/lib $NX_DIR/bin/nxdesktop -name "NX - $user@$HOSTNAME:$display - $session (GPL Edition)" $K $G $U $P $agent_server 2>>~/.nx/C-$sess_id/session &
 	else
 
 	# nxviewer session
@@ -171,10 +171,10 @@
 	then
 		mkdir -p ~/.nx/C-$sess_id/scripts/
 		echo "$agent_password" | $NX_DIR/bin/nxpasswd ~/.nx/C-$sess_id/scripts/.passwd doit
-		$NX_DIR/bin/nxviewer -encodings tight hextile copyrect raw -passwd ~/.nx/C-$sess_id/scripts/.passwd -name "NX - $user@$HOSTNAME:$display - $session (GPL Edition)"  $agent_server 2>>~/.nx/C-$sess_id/session &
+		LD_LIBRARY_PATH=$NX_DIR/lib $NX_DIR/bin/nxviewer -encodings tight hextile copyrect raw -passwd ~/.nx/C-$sess_id/scripts/.passwd -name "NX - $user@$HOSTNAME:$display - $session (GPL Edition)"  $agent_server 2>>~/.nx/C-$sess_id/session &
 	else
 	# "normal" nxagent session
-		$NX_DIR/bin/nxagent -persistent -name "NX - $user@$HOSTNAME:$display - $session (GPL Edition)" $K $G $B :$display 2>>~/.nx/C-$sess_id/session &
+		LD_LIBRARY_PATH=$NX_DIR/lib $NX_DIR/bin/nxagent -persistent -name "NX - $user@$HOSTNAME:$display - $session (GPL Edition)" $K $G $B :$display 2>>~/.nx/C-$sess_id/session &
 	fi
 	fi
 	PID=$!
--- nxkeygen.old	2004-10-14 18:57:21.000000000 +0200
+++ nxkeygen	2004-10-14 18:57:24.000000000 +0200
@@ -13,11 +13,7 @@
 #
 # ========================================================================
 
-# Get NX_ configuration variables
-export PATH=$PATH:$(cd $(dirname $0) && pwd)
-export $(grep ^NX_ $(which nxserver))
-
-NX_KEY_DIR="$NX_HOME_DIR/.ssh"
+NX_KEY_DIR="/usr/NX/home/nx/.ssh"
 DATE="`date '+%Y%m%d-%H%M%S'`"
 NX_CLIENT_KEY="${NX_KEY_DIR}/client.id_dsa.key"
 NX_SERVER_KEY="${NX_KEY_DIR}/server.id_dsa.pub.key"
@@ -49,9 +45,6 @@
 		chmod 600 $x
 		chown nx:root $x
 	done
-	
-	# copy the key to the authorized_keys2 file
-	cp -af ${NX_SERVER_KEY} $NX_HOME_DIR/$NX_AUTHORIZED_KEYS_FILE
 
 	# now tell the user what to do
 
