--- nxserver.old	2005-02-01 01:02:11.000000000 +0100
+++ nxserver	2005-02-01 01:04:19.000000000 +0100
@@ -13,7 +13,7 @@
 #
 
 # Important constants
-NX_CONFIG_FILE=/etc/nxserver/node.conf
+NX_CONFIG_FILE=/usr/NX/etc/node.conf
 
 # Read the config file
 . $NX_CONFIG_FILE
--- nxkeygen.old	2005-02-01 01:02:19.000000000 +0100
+++ nxkeygen	2005-02-01 01:04:39.000000000 +0100
@@ -53,9 +53,6 @@
 		chmod 600 $x
 		chown nx:root $x
 	done
-	
-	# copy the key to the authorized_keys2 file
-	cp -af ${NX_SERVER_KEY} $NX_HOME_DIR/$NX_AUTHORIZED_KEYS_FILE
 
 	# now tell the user what to do
 
--- node.conf.old	2005-02-01 01:02:27.000000000 +0100
+++ node.conf	2005-02-01 01:04:05.000000000 +0100
@@ -10,12 +10,12 @@
 # Path directives
 # These tells freeNX where various files is to be found
 
-NX_DIR=/usr
+NX_DIR=/usr/NX
 PATH_BIN=$NX_DIR/bin
 PATH_LIB=$NX_DIR/lib
-NX_ETC_DIR=/etc/nxserver
-NX_SESS_DIR=/var/lib/nxserver
-NX_HOME_DIR=/home/.nx
+NX_ETC_DIR=$NX_DIR/etc
+NX_SESS_DIR=$NX_DIR/var/db
+NX_HOME_DIR=$NX_DIR/home/nx
 
 # Logfile directives
 # Before turning logging on, please make sure that NX_LOGFILE is 
@@ -117,4 +117,4 @@
 NX_AUTHORIZED_KEYS_FILE=.ssh/authorized_keys2
 
 # adds LD_LIBRARY_PATH to the startup of nxagents
-NX_NOMACHINE_WAY=0
+NX_NOMACHINE_WAY=1
